src/matrix_input.jl

```diff
@@ -228,3 +229,80 @@ struct MILP{T, LP<:AbstractLPForm{T}}
     lp::LP
     variable_type::Vector{VariableType}
 end
+
+struct ConicForm{T, AT<:AbstractMatrix{T}, VT <: AbstractVector{T}} <: AbstractConicForm{T}
+    # assuming minimization for now
+    # direction::MOI.OptimizationSense 
+    c::VT
+    A::AT
+    b::VT
+    cones::Vector{<: MOI.AbstractVectorSet}
+end
+
+"""
+Convert a `MathOptInterface` model to `MatrixOptInterface` model
+"""
+function getConicForm(::Type{T}, model::M, con_idx) where {T, M <: MOI.AbstractOptimizer}
```

Comment:
Julia tends to use snake case, any specific reason here why not `get_conic_form`?

Reply:
The JuMP style guide recommends `conic_form` or `get_conic_form`

Reply:
thanks, replaced!